### DISCUSSION TITLE
Easy REPL-like way to evaluate NSS config syntax?

### DISCUSSION THREAD

**Discussion by stamminator** [moudey/Shell#498] [Q&A]
For example, if I want to compare the output of the `user.directory` and `user.profile` functions to make sure they're what I expect, I currently have to create dummy items somewhere, use function outputs as the titles (e.g. `item(mode="single" title=@user.directory)`), re-register NSS, then clean up and re-register again when I'm done.

Is there an easier way to run NSS code and evaluate the output? Perhaps a way to pass in statements to be evaluated via the command line? For example:

```cmd
> .\shell.exe --run "user.directory()"
:: outputs "C:\Users\jacob"
```

Something like this would make it easier to experiment with out setups and contribute to the project.

**Comment by moudey** [comment]
```js
item(title="test" cmd=msg('
    hello @user.name
    @user.directory()
'))
```
You do not need to re-register Shell. Just Save changes, 
press <kbd>CTRL</kbd> + <kbd>RIGHT-CLICK</kbd> 
or 
press <kbd>LEFT-CLICK</kbd>  + <kbd>RIGHT-CLICK</kbd> 
to reload settings.
